packages/gestalt/src/Avatar.js

```diff
@@ -60,6 +60,7 @@ type AvatarProps = {|
   size?: 'sm' | 'md' | 'lg',
   src?: string,
   verified?: boolean,
+  icon?: string,
```

Comment:
why is `icon` a string if down below you force it to be either `pinterest` or `check-circle`?

Reply:
@pfarejowicz  it was for test that was breaking initially but I changed it back to `'check-circle' | 'pinterest'`